internal/services/authorization/role_management_policy_resource.go

```diff
@@ -0,0 +1,1097 @@
+							state.EligibleAssignmentRules[0].ExpireAfter = rule.Values["maximumDuration"].(string)
+
+						case "Expiration_Admin_Assignment":
+							state.ActiveAssignmentRules[0].ExpirationRequired = rule.Values["isExpirationRequired"].(bool)
+							state.ActiveAssignmentRules[0].ExpireAfter = rule.Values["maximumDuration"].(string)
+
+						case "Expiration_EndUser_Assignment":
+							state.ActivationRules[0].MaximumDuration = rule.Values["maximumDuration"].(string)
+
+						case "Notification_Admin_Admin_Assignment":
+							state.NotificationRules[0].ActiveAssignments[0].AdminNotifications = []RoleManagementPolicyNotificationSettings{
+								*flattenNotificationSettings(pointer.To(rule.Values)),
+							}
+
+						case "Notification_Admin_Admin_Eligibility":
+							state.NotificationRules[0].EligibleAssignments[0].AdminNotifications = []RoleManagementPolicyNotificationSettings{
+								*flattenNotificationSettings(pointer.To(rule.Values)),
+							}
+
+						case "Notification_Admin_EndUser_Assignment":
+							state.NotificationRules[0].EligibleActivations[0].AdminNotifications = []RoleManagementPolicyNotificationSettings{
+								*flattenNotificationSettings(pointer.To(rule.Values)),
+							}
+
+						case "Notification_Approver_Admin_Assignment":
+							state.NotificationRules[0].ActiveAssignments[0].ApproverNotifications = []RoleManagementPolicyNotificationSettings{
+								*flattenNotificationSettings(pointer.To(rule.Values)),
+							}
+
+						case "Notification_Approver_Admin_Eligibility":
+							state.NotificationRules[0].EligibleAssignments[0].ApproverNotifications = []RoleManagementPolicyNotificationSettings{
+								*flattenNotificationSettings(pointer.To(rule.Values)),
+							}
+
+						case "Notification_Approver_EndUser_Assignment":
+							state.NotificationRules[0].EligibleActivations[0].ApproverNotifications = []RoleManagementPolicyNotificationSettings{
+								*flattenNotificationSettings(pointer.To(rule.Values)),
+							}
+
+						case "Notification_Requestor_Admin_Assignment":
+							state.NotificationRules[0].ActiveAssignments[0].AssigneeNotifications = []RoleManagementPolicyNotificationSettings{
+								*flattenNotificationSettings(pointer.To(rule.Values)),
+							}
+
+						case "Notification_Requestor_Admin_Eligibility":
+							state.NotificationRules[0].EligibleAssignments[0].AssigneeNotifications = []RoleManagementPolicyNotificationSettings{
+								*flattenNotificationSettings(pointer.To(rule.Values)),
+							}
+
+						case "Notification_Requestor_EndUser_Assignment":
+							state.NotificationRules[0].EligibleActivations[0].AssigneeNotifications = []RoleManagementPolicyNotificationSettings{
+								*flattenNotificationSettings(pointer.To(rule.Values)),
+							}
+						}
+					}
+				}
+			}
+
+			return metadata.Encode(&state)
+		},
+	}
+}
+
+func (r RoleManagementPolicyResource) Update() sdk.ResourceFunc {
+	return sdk.ResourceFunc{
+		Timeout: 5 * time.Minute,
```

Comment:
Should this timeout be longer, perhaps the same as the Create timeout?

```suggestion
		Timeout: 30 * time.Minute,
```